modules/learn/pages/security/roles.adoc

```diff
@@ -1943,6 +1943,59 @@ It allows the reading of bucket settings.
 ^| image:introduction/no.png[]
 |===
 
+
+[#query-sequential-scan]
+== Query Use Sequential Scan
+
+The *Query Use Sequential Scan* role (a Query & Index role) allows users to perform a sequential scan query.
+This is a query of a keyspace that does not have a suitable index. 
```

Comment:
The planner will only select a sequential scan as the access method when a suitable index is not available.